charts/document-engine/templates/storage-cleanup.CronJob.yaml

```diff
@@ -28,7 +28,7 @@ spec:
           labels:
           {{- include "document-engine.cleanupSelectorLabels" . | nindent 12 }}
           {{- with .Values.pspdfkit.storage.cleanupJob.podLabels }}
-          {{- toYaml . | nindent 12 }}
+            {{- toYaml . | nindent 12 }}
```

Comment:
How was this even working before?

Reply:
`{{-` eats whitespace to the left. It's a cosmetic change.